.travis.yml

```diff
@@ -205,3 +218,9 @@ script:
       bjam arm-tests warnings-as-errors=on ssl=$ssl crypto=$crypto variant=test_arm $toolset target-os=linux link=static testing.launcher="sudo cp -R bin rootfs/; sudo chroot rootfs";
     fi'
   - cd ..
+
+# test build for iOS, using x86_64 to avoid the need to build boost from source for arm
```

Comment:
I don't see where `BOOST_ROOT` is set, are you talking about the `if "$arch" == "arm"`? I will rephrase the comment, but basically, I'm using a x86_64 arch with the iOS simulation toolchain, to avoid the need of "arm boost build" (they are not provided by brew).